### PR TITLE
Culture Amp requiring WFH for all, 16/Mar/2020.

### PR DESCRIPTION
**Pull request by toothbrush**
Modifies the following events or companies:
 - Culture Amp

WFH required starting 16/Mar/2020 inclusive.  Announced Thu Mar 12 15:38 AEDT 2020.

### Checklist

 - [x] (N/A) I have updated the event or company counts
 - [x] I have put the most recent relevant date in the "Last Update" column
 - [ ] (announcement was on internal Slack) I have linked to the article about the change, not the company's website

### CODE REVIEW

**Comment by phildini** [comment]
Hi there! We recently re-worked the site to use yaml data files. We'd be happy to accept your PR once it's been reworked to use the right format.

Thanks!

**Comment by toothbrush** [comment]
Done!  Sorry @phildini, i'd totally missed the memo.

File: _data/companies.yml [hunk 1]
```diff
@@ -286,11 +286,11 @@
   last_update: 2020-03-05
 
 - name: Culture Amp
-  wfh: Encouraged
+  wfh: Restricted
```
Comment (drewbrew) [inline _data/companies.yml]:
```suggestion
  wfh: Required
```

Should that be required rather than restricted?

**Comment by toothbrush** [comment]
Geez you're right, sorry – it was late.

**Comment by toothbrush** [comment]
This should be ready for merging, thanks @drewbrew.

**Pull request merged**